lib/msal-browser/src/network/FetchClient.ts

```diff
@@ -29,14 +27,13 @@ export class FetchClient implements INetworkModule {
      * @param headers 
      * @param body 
      */
-    async sendPostRequestAsync(url: string, headers?: Map<string, string>, body?: string): Promise<any> {
+    async sendPostRequestAsync(url: string, headers?: Map<string, string>, reqBody?: string): Promise<any> {
```

Comment:
since the reqBody is optional here, and we're always making a post in this function, are there are weird scenarios/edge cases that can come up because of that?

Reply:
You have the option of sending a blank body in a POST.

Reply:
right, wanted to make sure it was a valid case, and it sounds like it is.